Source/Private/ASCollectionLayout.mm

```diff
@@ -140,8 +140,7 @@ - (void)invalidateLayout
 - (CGSize)collectionViewContentSize
 {
   ASDisplayNodeAssertMainThread();
-  ASDisplayNodeAssertNotNil(_layout, @"Collection layout state should not be nil at this point");
-  return _layout.contentSize;
+  return _layout ? _layout.contentSize : CGSizeZero;
```

Comment:
So that we don't regress on this in the future, could you add a link to this PR explaining the workaround?